demisto_sdk/commands/generate_test_playbook/test_playbook_generator.py

```diff
@@ -271,21 +278,57 @@ def create_automation_task_and_verify_outputs_task(test_playbook, command, item_
         test_playbook.add_task(task_verify_outputs)
 
 
+def get_command_examples(commands_file_path) -> dict:
+    """
+    Gets the command examples from command file with their arguments.
+
+    Args:
+        commands_file_path: command file or the content of such file.
+        specific_commands: commands specified by the user.
+
+    Return:
+        dict. Arguments separated by the commands.
+    """
+    command_examples = []  # type: list
+
+    if os.path.isfile(commands_file_path):
```

Comment:
If I recall correctly `isfile` does not only check the validity of the path it also checks its existence.
so in case you are given an integration and the command examples file path is not real, it will go to the else. and this can be weird...

might be wiser to check the file type and by that chose which flow should run.